extra/utils/Validation.js

```diff
@@ -0,0 +1,57 @@
+let users = [
+    {
+        traineeEmail : 'shashank.baranawal@successive.tech',
+        reviewerEmail : 'naman.parashar@successive.tech',
+    },
+    {
+        traineeEmail : 'shashank.baranwal@gmail.com',
+        reviewerEmail : 'naman.parashar@gmail.com'
+    }
+]
+
+/*function validateEmail(email){
+    const regix = /\w+.\w+@successive.tech$/i;
+    return regix.test(email)
+}*/
+
+function validateEmail(email){
+    const regex = /\w+.\w+@successive.tech$/i;
+    if(regex.test(email))
+    {
+        return true;
+    }
+    else{
+        return false;
+    }
+}
+//console.log(validateEmail(users[0]["traineeEmail"]))
+let a =[];
+let b =[];
+function validateUsers(users){
+    let countValid=0;
+    let countInvalid=0;
+    users.forEach(item => {
+        const {traineeEmail,reviewerEmail}= item;
+        if(validateEmail(item.traineeEmail)){
+            a.push(traineeEmail);
+            countValid++;
```

Comment:
Still using dot method to access object values

Reply:
Redundant access of object values

Reply:
i corrected all the things